runtimes/php/tutorial-laravel.md

```diff
@@ -98,3 +98,30 @@ Create a new environment variable called `CC_FS_BUCKET` and set `/storage/app:<b
 If you need to build your frontend assets (eg. javascript or CSS files), you can either add it as a step in your composer file, or you can add a post build hook with the `CC_POST_BUILD_HOOK` environment variable.
 
 For example, if you launch the build with `npm run prod`: `CC_POST_BUILD_HOOK=npm install && npm run prod`.
+
+### Optional: configure task scheduling
+
+If your app uses [task scheduling](https://laravel.com/docs/scheduling), you need to configure a cron:
```

Comment:
Maybe adding why you need to configure a cron might help? Like `you need to configure a cron to start the scheduling process`?